web/src/components/atoms/DownloadButton/index.tsx

```diff
@@ -19,7 +19,16 @@ const DownloadButton: React.FC<DownloadButtonProps> = ({
   ...props
 }) => {
   const handleDownload: MouseEventHandler<HTMLElement> | undefined = () => {
-    fileDownload(url, filename);
+    fetch(url, {
+      method: "get",
```

Comment:
method `get` should be upper-case by convention. (to be `GET`)

Reply:
Done!